roles/ceph-validate/tasks/check_devices.yml

```diff
@@ -2,15 +2,16 @@
 - name: devices validation
   when: devices is defined
   block:
-    - name: validate devices is actually a device
+    - name: get devices information
       parted:
         device: "{{ item }}"
         unit: MiB
       register: devices_parted
+      failed_when: False
```

Comment:
wondering what's the best way to do that compared to using `ignore_errors: true` instead of `failed_when: false`.
With `ignore_errors: true` you don't have to change the next task.
Any thoughts ?